test/appium/tests/atomic/account_management/test_profile.py

```diff
@@ -33,20 +33,80 @@ def test_mobile_data_usage_popup_continue_syncing(self):
         sign_in_view.create_user()
         sign_in_view.just_fyi("Enable mobile network to see popup and enable syncing")
         sign_in_view.toggle_mobile_data()
-        if not sign_in_view.find_text_part("Sync using Mobile data"):
+        if not sign_in_view.element_by_text_part("Sync using Mobile data").is_element_displayed():
             self.driver.fail('No popup about Mobile data is shown')
         sign_in_view.wait_for_element_starts_with_text('Continue syncing').click()
 
-        sign_in_view.just_fyi("Checked that selected option is stored in Profile")
+        sign_in_view.just_fyi("Check that selected option is stored in Profile")
         profile_view = sign_in_view.profile_button.click()
         profile_view.sync_settings_button.click()
         profile_view.element_by_text('Mobile data').click()
-        if not profile_view.use_mobile_data.attribute_value('checked'):
-            self.errors.append('"Use mobile data" is disabled')
-        if not profile_view.ask_me_when_on_mobile_network.attribute_value('checked'):
-            self.errors.append('"Ask me when on mobile network" toggle is disabled')
+        for toggle in profile_view.use_mobile_data, profile_view.ask_me_when_on_mobile_network:
+            if not toggle.attribute_value('checked'):
+                self.errors.append("Toggles in Mobile settings are not enabled")
+
+        sign_in_view.just_fyi("Check that can join public chat ans send message")
+        chat_name = sign_in_view.get_public_chat_name()
+        home = profile_view.get_back_to_home_view()
+        chat = home.join_public_chat(chat_name)
+        message = 'test message'
+        chat.chat_message_input.send_keys(message)
+        if not chat.chat_element_by_text(message).is_element_displayed():
+            self.errors.append("Message was not sent!")
```

Comment:
I'd check here the message status after it sent rather than message itself

Reply:
It makes more sense on IOS, on Android you can't even send a message from offline. 
But I can add check

Reply:
also please add self.verify_no_errors() in the end of the test

Reply:
thx, added.